docs/advanced-usage/multi-domain.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Multi domain setup
+description: Learn how to achieve a multi domain setup in order to share authentication and session management across domains.
+---
+
+import { Callout, Steps} from "nextra-theme-docs";
+import { Images } from "@/components/Images";
+import { CodeBlockTabs, Tabs, Tab } from "@/components/CodeBlockTabs";
+
+# Multi domain setup
+
+Clerk supports sharing sessions across multiple domains with a primary and satellite configuration.
+
+Having multiple domains for your instance enables you to have central authentication and session management across all of them.
+
+There is always a primary domain, but there can be many satellite domains.
+
+The primary domain is the source of truth for the user's authentication state. Users must complete the sign-in flow on this domain, either using Clerk’s `<SignIn/>` component or our `useSignIn()` hook.
+
+As for satellite domains, When users initially request one, they are transparently redirected through the primary domain to determine the authentication state. If users want to sign-in, they must be redirected to a sign-in flow hosted on the primary domain, then redirected back.
+
+## How to set up a multi domain project?
+
+<Steps>
+
+### Create your application and install Clerk
+Build your application just like you would any other application with Clerk. Currently, multi-domain can be added to any Next.js or Remix application. For other React frameworks, multi-domain is still supported as long as you do not use server rendering or hydration.
+
+Once you create an instance via the dashboard, you will be prompted to choose a domain. We consider this to be the primary domain.
+
+We will be using the domain `primary.dev` for the purposes of this guide.
+
+<Callout type="info">
+Note: For multi-domain, you must configure your sign-in flow to run within your primary application, for example on `/sign-in`.
+</Callout>
+
+### Configure your primary app
+
+Configuration for this app is the same as for any other app using Clerk without having multiple domains.
+
+<CodeBlockTabs options={["Next.js", "Remix"]}>
+```env copy filename=".env.local"
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_live_...
+CLERK_SECRET_KEY=sk_live_...
+```
+```env filename=".env" copy
+CLERK_PUBLISHABLE_KEY=pk_live_...
+CLERK_SECRET_KEY=sk_live_...
+```
+</CodeBlockTabs>
+
+### Register your satellite domain
+
+Go to the "Domains" tab in the Clerk dashboard and add a new satellite domain of your choice. Then, follow the instructions provided.
```

Comment:
We use **bold** lettering instead of "quotes" when guiding a user through a website. See any of the pages on /docs/authentication/social-connections/[page, e.g. google]

```suggestion
Go to the **Domains** tab in the Clerk dashboard and add a new satellite domain of your choice. Then, follow the instructions provided.
```